mp/importer/batcher.py

```diff
@@ -0,0 +1,40 @@
+
+def run_in_batches(iterable, end_batch_callback=None, batch_size=2, batch_start=0, max_batches=None):		# what if max_batches = None? TODO
+    c = 0
+    ended = False
+    if max_batches is None:
```

Comment:
This is a lot of code duplication for a very simple condition. Try push it down into the loop:

{{{
if current_batch >= batch_start and (max_batches is None or current_batch < batch_start+max_batches):
}}}